docs/index.md

```diff
@@ -145,9 +140,57 @@ illustrates how to do this:
     	$ unset DOCKER_TLS_VERIFY
       ```
 
+## Discovery (optional)
```

Comment:
"Setting up a Discovery service" (or similar?)

Not sure if the "(optional)" should be added; I "get" why you added it, but if the introduction text is clear enough, it may be redundant.

Perhaps repeat here that in most cases, a comma-separated list is not the recommended way, and (I think) mainly for testing/development setups.